### PR TITLE
Update charts

### PR DESCRIPTION
**Pull request by etaques**
This PR changes:
- Update dependency charts due bitnami changes on its repository.

### CODE REVIEW

**Comment by weyrick** [comment]
Why would we upgrade these? These are major version of grades which will require testing.

**Comment by etaques** [comment]
Do you prefer we change to the minimal version number?

**Comment by etaques** [comment]
Added the minimal version number available on helm chart dependencies.

File: charts/orb/Chart.yaml [hunk 1]
```diff
@@ -25,50 +25,50 @@ dependencies:
     repository: "@jaegertracing"
     condition: jaeger-operator.enabled
   - name: nats
-    version: "6.4.2"
+    version: "7.0.4"
```
Comment (weyrick) [inline charts/orb/Chart.yaml:28]: still major version, they don't have anything in 6.x? if not, and we must go to 7.x, then we have to test first.

**Pull request closed without merging**